src/kernel/include/shared/cos_types.h

```diff
@@ -217,6 +224,7 @@ __captbl_cap2sz(cap_t c)
 	case CAP_TCAP:
 		return CAP_SZ_16B;
 	case CAP_HW: /* TODO: 256bits = 32B * 8b */
+	case CAP_ULK:
```

Comment:
@evanstella @gparmer I suggest we make its size to be 64B because of the cacheline contention problem.

Reply:
@betahxy Mind explaining a little more? I don't really get y we might run into cache line contention problem. Is that x64 specific?